packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -70,5 +70,52 @@ void main() {
     expect(find.text('B'), findsOneWidget);
     box = tester.renderObject(find.byType(ExpansionPanelList));
     expect(box.size.height - oldHeight, greaterThanOrEqualTo(100.0)); // 100 + some margin
+
+    // multiple panel list test
```

Comment:
you should put this in a separate testWidgets call, with this comment as the title of the test